FccSoftwareGit.md

```diff
@@ -47,66 +63,70 @@ You will be using (at least) 3 versions of the FCCSW repository:
 The repositories 1 and 2 are added as remote to the repository 3:
 
 ```bash
-git clone git@github.com:[YOUR_GITHUB_USER]/FCCSW.git
+git clone git@github.com:[YOUR_GITHUB_USER]/FCCSW.git # create a local copy 1 of your fork 2
```

Comment:
Shouldn't that be "local copy 3" ?
I would also add parentheses to '1' '2' and '3' make it consistent with below.